RELEASENOTES.md

```diff
@@ -14,6 +14,51 @@ release) was only available in one of the two variants, we note that
 in the release notes.
 
 
+# v1.7.4-aws release notes
+This release requires [Libfabric v1.11.0](https://github.com/ofiwg/libfabric/releases/tag/v1.11.0)
+or later and supports [NCCL v2.19.3-1](https://github.com/NVIDIA/nccl/releases/tag/v2.19.3-1) while
+maintaining backward compatibility with older NCCL versions ([NCCL v2.4.8](https://github.com/NVIDIA/nccl/releases/tag/v2.4.8-1) and later).
+It was tested with Libfabric versions up to
+[Libfabric v1.19.0](https://github.com/ofiwg/libfabric/releases/tag/v1.19.0).
+
+With NCCL 2.18.5 or later and v1.7.3-aws of the plugin,
```

Comment:
v1.7.3-aws or later? Do we still need this paragraph at all? It is the same in the 1.7.3-aws release notes.

Reply:
fixed